starter/graphql/schema.graphqls

```diff
@@ -0,0 +1 @@
+
```

Comment:
I had this here by default because I didn't think there was a lot of value in having a step in the tutorial to add this file to the graphql folder in xcode after they fetch it, although to the same point they dont necessarily need to add it there at all, thats just how/where the tutorial has the user adding .graphql files

Reply:
OK, I got the intent. That's fine then. It gives them visibility within Xcode to be able to view the schema.